### ISSUE TITLE
Runpod - How to add Lora without link in Stable Diffusion ?

### ISSUE THREAD

**Issue by HelpANewbie** [TheLastBen/fast-stable-diffusion#2869]
In folders - where do we need to upload the LORA files ? I want to add multiple LORAs - Can someone clearly define the path to the LORA adding folder ? 

I went in extentions and added there but it doesn't show up when I run SD webui. 

Please help.

**Comment by TheLastBen** [comment]
the lora folder is `/workspace/sd/stable-diffusion-webui/models/Lora`

**Comment by HelpANewbie** [comment]
> the lora folder is `/workspace/sd/stable-diffusion-webui/models/Lora`

Thanks so much Ben :)

**Comment by HelpANewbie** [comment]
> the lora folder is `/workspace/sd/stable-diffusion-webui/models/Lora`

Hi Ben - When I load the models in this folder they dont show up in the webui. Only when I put a drive link in Lora "link", it shows in the webui after loading. And in the Lora link - i can put only one link right ? is there a way i can add more links in the Lora link part so it loads multiple Loras ?

**Comment by HelpANewbie** [comment]
> > the lora folder is `/workspace/sd/stable-diffusion-webui/models/Lora`
> 
> Hi Ben - When I load the models in this folder they dont show up in the webui. Only when I put a drive link in Lora "link", it shows in the webui after loading. And in the Lora link - i can put only one link right ? is there a way i can add more links in the Lora link part so it loads multiple Loras ?

Anyone can help on this please ?

**Comment by TheLastBen** [comment]
Make sure you're putting the lora inside the right folder, use a link first, then check the right folder where it was downloaded to, then use that folder for other loras

**Comment by HelpANewbie** [comment]
Yes i did use the right folder and the after I upload the additional Lora - it doesn't show up in the Sd webui - am I supposed to restart the webui or something like that ?

**Comment by TheLastBen** [comment]
yes, try restarting

**Comment by HelpANewbie** [comment]
I refreshed the webui page but it doesnt show...Do i run the code from start to restart ? wont the loaded loras go away then ?

**Comment by HelpANewbie** [comment]
So I loaded the Lora in the same folder but when I refresh then the new LORA doesn't show in the webui. 

I also tried refreshing the link. Also tried running the code from the Lora part and the Load Model part but it doesn't run. When I do that then nothing loads so I have to end the pod and then restart the whole procedure to get it running with only one Lora as earlier.

Is there any way I can add multiple Loras by putting the links of the drive there so it automatically loads up in the Lora folder ?
